frontend/src/components/viewers/Tensorboard.tsx

```diff
@@ -56,6 +64,10 @@ class TensorboardViewer extends Viewer<TensorboardViewerProps, TensorboardViewer
     this._checkTensorboardApp();
   }
 
+  public onChangeFunc(e: React.ChangeEvent<{ name?: string; value: unknown }>): void {
```

Comment:
Behavior issue: after a version is selected, BusyButton state is no longer up-to-date, because it checks if the selected version instance exists or not.

I think the behavior could be confusing.
My imagined ideal behavior:
1. backend finds tensorboard instance by logdir and reports which version it is running
2. frontend shows to user there is an existing tensorboard with version xxx
3. user can choose to stop it and start a new one with different version

What do you think? Feel free to say your opinion. We discuss to make the product.

Reply:
Yes, I agree. Since stopping and restarting a tensorboard requires a delete button, we plan to add the delete feature in next commit and the behavior will be completed as you mentioned above at that time.